### ISSUE TITLE
SIGTRAP on launch Debian 10

### ISSUE THREAD

**Issue by mryellow** [microsoft/vscode#126148]
- VS Code Version: 

```
1.57.0
b4c1bd0a9b03c749ea011b06c6d2676c8091a70c
x64
```

- OS Version:

```
Linux debian 4.19.0-16-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19) x86_64 GNU/Linux
```

Steps to Reproduce:

1. `code --status --disable-extensions`
2. `[9718:0613/092950.820177:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /snap/code/66/usr/share/code/chrome-sandbox is owned by root and has mode 4755.`

Or without `--status` `dmesg` shows:

```
[ 1980.058806] traps: code[9718] trap int3 ip:565518ca4f28 sp:7ffe9a7d1010 error:0 in code[565518a75000+5dfa000]
```

**Comment by mryellow** [comment]
```
ls -la /snap/code/66/usr/share/code/chrome-sandbox
-rwxr-xr-x 1 root root 4712840 Jun 10 03:39 /snap/code/66/usr/share/code/chrome-sandbox
```

**Comment by mryellow** [comment]
Worked around by removing the Snap version and installed `codium` from repo.

```
sudo snap remove --purge code
```

https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo

**Comment by msanguineti** [comment]
I can confirm as well (`code` installed with `snap install --classic code`)

```
Version: 1.57.0
Commit: b4c1bd0a9b03c749ea011b06c6d2676c8091a70c
Date: 2021-06-09T17:18:42.895Z
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Linux x64 4.19.0-16-amd64 snap
```

```sh
$ code --verbose
[8686:0617/122132.316034:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /snap/code/66/usr/share/code/chrome-sandbox is owned by root and has mode 4755.
```

I managed to start `code` by:

```sh
$ code --no-sandbox
Warning: 'sandbox' is not in the list of known options, but still passed to Electron/Chromium.
```
`code` starts despite the warning.

**Comment by deepak1556** [comment]
Sorry for the breakage, this should be fixed in `1.57.1`. Merging to https://github.com/microsoft/vscode/issues/126007
